css/index.css

```diff
@@ -0,0 +1,79 @@
+@media screen and (min-width: 480px) {
+
+
+	html {
+		display: flex;
+		flex-flow: row wrap;
+		justify-content: space-around;
+	
+		background-color: black;
+		font-family: "Times New Roman", Times, serif;
+		color: green;
+		padding: 1px;
+		margin: 1px;
+	}
+}
+
+header {
+    background-color: gray;
+    color: white;
+    padding: 1px;
+}
+
+body {
+    background-color: #f8f8ff;
+}
+
+section {
+
+}
+
+section ul {
+
+}
```

Comment:
One of the tasks was to add padding/margin between sections. To do so in CSS you would add those properties here!